### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 10th of January 2023 17:30 CET / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#934] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 10th of January 2023 17:30 CET / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [here](https://youtu.be/e8QVlOyfa5w)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

### News
- Mysterious link: https://github.com/cncf/toc/pull/991

### Spotlight
- Presentation: Demo Flatcar CAPZ deployment from Community Gallery by @invidian 

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

**Comment by invidian** [comment]
Is this for 2023?

**Comment by invidian** [comment]
Can we add:
- Presentation: Demo Flatcar CAPZ deployment from Community Gallery

To the agenda please?
